src/realm/query_conditions.hpp

```diff
@@ -907,21 +918,24 @@ struct Less {
         return Mixed::types_are_comparable(m1, m2) && (m1 < m2);
     }
 
+    bool operator()(int64_t v1, int64_t v2) const
+    {
+        return v1 < v2;
+    }
+
     template <class A, class B, class C, class D>
     bool operator()(A, B, C, D) const
     {
         REALM_ASSERT(false);
         return false;
     }
     static const int condition = cond_Less;
-    bool can_match(int64_t v, int64_t lbound, int64_t ubound)
+    bool can_match(int64_t v, int64_t lbound, int64_t)
```

Comment:
Can we make this method a 2 parameter method? Instead of having 3 params.

Reply:
Not really. some of the implementations use the other parameter. The compiler will optimize.